src/components/StakingTimeline/index.tsx

```diff
@@ -81,7 +78,7 @@ export const LegacyStakesDepositSteps: FC = () => {
             Enter the Provider, Beneficiary, and Authorizer addresses. These
             will be automatically set to your wallet address. If you want to use
             a Staking Provider, here is{" "}
-            <InternalLink to={STAKING_PROVIDER_URL}>a list</InternalLink>
+            <Link to={STAKING_PROVIDER_URL}>a list</Link>
```

Comment:
Non-blocking:

Do you think we should add dot here at the end?

```suggestion
            <Link to={STAKING_PROVIDER_URL}>a list</Link>.
```